index.js

```diff
@@ -0,0 +1,422 @@
+let buttons = [
+  { which: 192, code: "Backquote", en: "`", ru: "ё", shift: "~", ruShift: "Ё" },
```

Comment:
I know that we discussed it earlier and there is no reason to change it at this point, but in the future consider to use [code](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code) as `which` [is being deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/which)

I don't know the reasons why `which` is being deprecated, but in my opinion code that uses `which` loses its readability - you know, there is tons of magic numbers, it's hard to gasp what's happening with them sometimes :sweat_smile: